internal/anclaHelper/anclaHelper.go

```diff
@@ -13,48 +11,23 @@ import (
 	"go.uber.org/zap"
 )
 
-type AnclaListenerIn struct {
+type In struct {
 	fx.In
-	Measures ancla.Measures
-	Logger   *zap.Logger
-}
-type AnclaServiceIn struct {
-	fx.In
-	Config   ancla.Config
+	Svc      *ancla.ClientService
 	Listener ancla.ListenerConfig
 	Sink     sink.Wrapper
+	LC       fx.Lifecycle
 }
 
-func InitializeAncla(lifecycle fx.Lifecycle) fx.Option {
-	return fx.Provide(
-		func(in AnclaListenerIn) ancla.ListenerConfig {
```

Comment:
moved to ancla (https://github.com/xmidt-org/ancla/pull/236/files#diff-c52daedaeda258db15ec11d57bbf299412a20328f6cd026980ef07f01960dd3aR189-R219)

Reply:
the provide functions are being called: https://github.com/xmidt-org/caduceus/pull/526/files#diff-305593783b1c45f15c0e2812db09596d4d8e8e52ae6403a36dbae45e54bcbfb3R161-R162